### PR TITLE
Plugin icons and description changed

### PR DESCRIPTION
**Pull request by DivyanshiSingh**
Fixes: #102 
I have updated the landing page with 5 store plugins available. 
I have changed the icons and the text in the category section (visualization) but not in all categories. Should I change other categories also with the same plugins? Or how can I know the plugins for the respective categories? @mairin @jennydaman

### CODE REVIEW

**Comment by DivyanshiSingh** [comment]
@mairin please review this and let me know if changes are required

**Comment by jennydaman** [comment]
There is a second issue being worked on which aims to get data dynamically.

https://github.com/FNNDSC/ChRIS_store_ui/issues/125

File: src/components/Welcome/components/WelcomeCTA/WelcomeCTA.js [hunk 1]
```diff
@@ -24,11 +20,11 @@ const WelcomeCTA = () => (
         Apps available in the ChRIS store include:
       </div>
       <div className="row welcome-cta-featured-container">
-        <WelcomeFeature name="FreeSurfer" img={freesurferLogo} />
-        <WelcomeFeature name="TensorFlow" img={tensorflowLogo} />
-        <WelcomeFeature name="FMRIB Software Library" img={fslLogo} />
-        <WelcomeFeature name="RhoANA" img={rhoanaLogo} />
-        <WelcomeFeature name="Slicer" img={slicerLogo} />
+        <WelcomeFeature name="pl-z2labelmap" img={freesurferLogo} />
```
Comment (jennydaman) [inline src/components/Welcome/components/WelcomeCTA/WelcomeCTA.js:23]: It'd be better to use human-friendlier titles, e.g. "Z-Score Map"

File: src/components/Welcome/components/WelcomeCategories/sampleCategories.js [hunk 2]
```diff
@@ -10,33 +10,34 @@ import ielectrodesLogo from '../../../../assets/img/plugins/ielectrodes_90.png';
...
+        desc: 'z-score to FreeSurfer label map',
         tags: ['TAG1', 'TAG2'],
       },
       {
-        img: brainbrowserLogo,
-        name: 'BrainBrowser',
-        desc: 'Lorem ipsum dolor',
+        img: freesurferLogo,
+        name: 'pl-dsdircopy',
+        desc: 'A ChRIS ds app to copy obj storage directories',
```
Comment (jennydaman) [inline src/components/Welcome/components/WelcomeCategories/sampleCategories.js:28]: Not really for visualization

File: src/components/Welcome/components/WelcomeCategories/sampleCategories.js [hunk 3]
```diff
@@ -10,33 +10,34 @@ import ielectrodesLogo from '../../../../assets/img/plugins/ielectrodes_90.png';
...
+        name: 'pl-fetal-brain-mask',
+        desc: 'Automatic masking of fetal brain images',
         tags: ['TAG1', 'TAG2'],
       },
       {
-        img: mricrosLogo,
-        name: 'MRIcroS',
-        desc: 'Lorem ipsum dolor',
+        img: freesurferLogo,
+        name: 'pl-flip',
```
Comment (jennydaman) [inline src/components/Welcome/components/WelcomeCategories/sampleCategories.js:39]: this plugin seems like a test plugin uploaded by some student.

https://github.com/zongxinc/pl-flip

**Review by jennydaman — COMMENTED** [review]
As aforementioned, the featured plugins should perhaps be fetched dynamically based on star count. https://github.com/FNNDSC/ChRIS_store_ui/issues/125

For a curtailed list of exemplar plugins, we should consult with the science-focused developers in our group (I'm one of them). 

`pl-fetal-brain-mask` does do visualization, and is a great plugin for this category. But it belongs to the "Segmentation" category instead. There is notable overlap between Segmentation and Visualization categories, as image segmentation is visual analysis.

**Comment by jennydaman** [comment]
Example for how to query the API for plugins by category:

```
https chrisstore.co/api/v1/plugins/search/ accept:application/json category==Segmentation | jq -r '.results[] | .name'
```

note: categories are self-defined by plugin developer. We take inspiration from https://www.nitrc.org/ but there are no formal categories. I don't think it was a good back-end design choice. I would also vouch for "Categories" to be an editable field of a plugin.

File: src/components/Welcome/components/WelcomeCategories/sampleCategories.js (hunk 2 above)
Comment (jennydaman) [reply src/components/Welcome/components/WelcomeCategories/sampleCategories.js:28]: https://chrisstore.co/plugin/71

Brain mask for in-utero MRI

https://chrisstore.co/plugin/61

Z score label map

Sorry, don't have screenshots for these two atm.
- https://chrisstore.co/plugin/66
- https://chrisstore.co/plugin/52


This issue speaks to a previous discussion we were talking about how "category" might not be as-useful as maybe something like "tags"

**Comment by mairin** [comment]
@jennydaman Do you know if the "Categories" are implemented as tags? It seems like some tools might fit into more than one category (is that correct?) so maybe having them set up to be tags so more than one could apply to a given plugin would make sense. What do you think?

**Comment by jennydaman** [comment]
@mairin In several disorganized threads, I've discussed how "tags" could be more useful than "Categories". Currently, tags are not implemented and it is one category per plugin.

**Comment by jennydaman** [comment]
"Tags" would require backend support.

**Comment by mairin** [comment]
@jennydaman Oh ok I wouldn't hold up this PR on that then, should we make a new issue for that on the ChRIS store backend?

**Comment by jennydaman** [comment]
Sure

**Comment by mairin** [comment]
https://github.com/FNNDSC/ChRIS_store/issues/43 for that

**Comment by DivyanshiSingh** [comment]
Can you please specify the changes that I have to do for this particular PR @jennydaman @mairin?

**Comment by jennydaman** [comment]
@DivyanshiSingh awaiting @mairin 's response to https://github.com/FNNDSC/ChRIS_store_ui/issues/125#issuecomment-819930288

**Comment by mairin** [comment]
> @DivyanshiSingh awaiting @mairin 's response to [#125 (comment)](https://github.com/FNNDSC/ChRIS_store_ui/issues/125#issuecomment-819930288)

Just dropped it, sorry for the wait

**Comment by jennydaman** [comment]
Closing due to preference for a solution to #150

**Pull request closed without merging**